.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,24 @@
+name: algolia-scraper
+on:
+  push:
+    branches:
+      - main
+  check_suite:
+    types: [completed]
+  workflow_dispatch:
+jobs:
+  scrape:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out code 🛎
+        uses: actions/checkout@v2
+      - name: scrape the site 🧽
+        env:
+          APPLICATION_ID: ${{ secrets.APPLICATION_ID }}
```

Comment:
Can you explain what will need to happen here?  We obviously need to add these keys to the repo.  What are they, how do we find them, etc?

Reply:
1. you would need to sign up on www.aligolia.com and create a project for the purpose of creating index
2. add an index_name called "docusaurus" for instance
3. you add API_key and give rights for ACLs search, addObject, deleteObject, editSearch 
4. site_url is the deployed site, but siteindex needs to match to crawl it
5. if u look at the pull request ff0a9a5 i have a sample .env file with the variables set.  I removed it from the next PR and replaced it with process.env. - before it was hardcoded.  Ideally you want this public so u can dev but I think each dev should just create his own account because its free and owner of repo just create secrets for it then it would work

Reply:
Please put this into a section on `Search` in the `CONTRIBUTING.md` file, so we don't lose track of it.  There's no way we'll remember all this.

Reply:
i added comments with sample .env so u can see its approximate values fc91f68 so someone can easily follow up

Reply:
> Please put this into a section on `Search` in the `CONTRIBUTING.md` file, so we don't lose track of it. There's no way we'll remember all this.

Added to CONTRIBUTING.md